doc.go

```diff
@@ -710,6 +711,10 @@ If enabled, directories will also be passed to the previewer script. This allows
 
 Draw boxes around panes with box drawing characters.
 
+	dupfilefmt        string      (default '%f%e.~%n~')
+
+Format string of file name when creating duplicate files. With the default format, copying a file `abc.txt` to the same directory will result in a duplicate file called `abc.txt.~1~`.
```

Comment:
You might want to clarify that the extension dot is part of the `%e` placeholder itself, it might not be immediately obvious to users. Maybe it would be good to just explicitly describe what each placeholder means, similar to the `promptfmt` option.

Reply:
Good idea! Fixed!